### ISSUE TITLE
ExifTag.TimeZoneOffset is defined as uint[], but the TIFF spec defines it as signed short

### ISSUE THREAD

**Issue by rwg0** [SixLabors/ImageSharp#2437]
### Prerequisites

- [X] I have written a descriptive issue title
- [X] I have verified that I am running the latest version of ImageSharp
- [X] I have verified if the problem exist in both `DEBUG` and `RELEASE` mode
- [X] I have searched [open](https://github.com/SixLabors/ImageSharp/issues) and [closed](https://github.com/SixLabors/ImageSharp/issues?q=is%3Aissue+is%3Aclosed) issues to ensure it has not already been reported

### ImageSharp version

3.0.1

### Other ImageSharp packages and versions

?

### Environment (Operating system, version and so on)

Windows 10 x64

### .NET Framework version

.NET 7

### Description

The TimeZoneOffset Exif tag is defined in the code as a uint[] (https://github.com/SixLabors/ImageSharp/blob/a49862a755fb24804043390f391839416bf4547a/src/ImageSharp/Metadata/Profiles/Exif/Tags/ExifTag.LongArray.cs#L62)

However, the specifications for TIFF/EP which defines that tag seem to require a signed short (optionally two signed shorts). See Page 19 and 40 of the TIFF/EP spec : https://web.archive.org/web/20060212163025/http://www.map.tu.chiba-u.ac.jp/IEC/100/TA2/recdoc/N4378.pdf

This requirement is also encoded in the TIFF verification code in JHove, which then flags the file as non-conformant (https://jhove.openpreservation.org/modules/tiff/tags/ , https://jhove.openpreservation.org/)

As a side note, since ExifTag<T> doesn't have a public constructor, I cannot create my own tags to work around this issue (or to add custom tags to the file).

### Steps to Reproduce

Set the DateTimeOffsetTag in a tiff file, then save the file

```
var image = Image.Load(sourceTifFile);
image.Frames[0].Metadata.ExifProfile = new ExifProfile();
image.Frames[0].Metadata.ExifProfile.SetValue(ExifTag.TimeZoneOffset, new uint[] {1});
image.Save(outputTifFile);
```

Now run the output tiff file through jhove verification

```
robin@ubuntu:~$ jhove -m TIFF-hul Output.tif 
Jhove (Rel. 1.20.0, 2019-01-19)
 Date: 2023-04-13 13:33:05 PDT
 RepresentationInformation: Output.tif
  ReportingModule: TIFF-hul, Rel. 1.8 (2017-05-11)
  LastModified: 2023-04-13 13:27:13 PDT
  Size: 343128
  Format: TIFF
  Status: Not well-formed
  SignatureMatches:
   TIFF-hul
  ErrorMessage: Type mismatch for tag 34858; expecting 8, saw 4
  MIMEtype: image/tiff

```

You can install jhove on an ubuntu box  with

`sudo apt-get install jhove default-jre`

Note: I ran the ImageSharp test code on Windows, just used Ubuntu for testing as the jhove install is easier

**Comment by dlemstra** [comment]
That looks like a mistake indeed but changing this would introduce a breaking change. But I do wonder if anyone is using this since we implemented this incorrectly. Maybe we could patch this in a minor release and make a note about this minor breaking change in the release notes?

**Comment by rwg0** [comment]
To avoid a breaking change you could attribute the existing TimeZoneOffset property as [Obsolete] and add a new property with the correct definition.

**Comment by JimBobSquarePants** [comment]
While it affects the public API I don’t consider bug fixes like this to require minor versions. I’d be happy shipping a fix without one.

**Comment by JimBobSquarePants** [comment]
I'm happy for this to be fixed in v3.1
